swan-lake/learn/testing-ballerina-code/writing-tests.md

```diff
@@ -282,6 +282,153 @@ function foo() {
 }
 ```
 
+### **Difference between expected and actual values when using 'assertEquals'**
```

Comment:
```suggestion
### **Difference between expected and actual values when using 'assertEquals'**
```
Titles should not be in bold.

Reply:
Ok noted, we might have to change all the other titles in this page too. Will do.